src/lib.rs

```diff
@@ -1,570 +1,11 @@
-//! libftdi1 is a library for working with FTDI chips like
-//! FT232BM, FT245BM, FT2232C, FT2232D, FT245R, FT232H, FT230X
-//! The documentation for it is available at http://www.intra2net.com/en/developer/libftdi/documentation/
+//! libftdi1 is a library for working with FTDI chips like FT232BM, FT245BM,
+//! FT2232C, FT2232D, FT245R, FT232H, FT230X The documentation for it is
+//! available at http://www.intra2net.com/en/developer/libftdi/documentation/
 //!
-//! This wrapper is based on the rust-bindgen output for libftdi 1.2
+//! This wrapper is based on the rust-bindgen output for libftdi 1.4
```

Comment:
This wrapper is not "based on", but generated with bindgen (the previous one, I believe, was manually edited after generation).

Reply:
It looked like it was manually edited, but I couldn't confirm or not. The options I passed to bindgen generated bindings that looked the _closest_ to what's in the repo. Will fix.